stable/sealed-secrets/templates/psp.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "sealed-secrets.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "sealed-secrets.name" . }}
+    helm.sh/chart: {{ template "sealed-secrets.chart" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+spec:
+  allowedCapabilities:
+  - '*'
+  fsGroup:
+    rule: RunAsAny
+  privileged: true
```

Comment:
This is way to permissive. I see no reason why sealed-secrets would need privileged, hostPID, hostIPC, hostNetwork.

Reply:
I know that's why I said permissive. I don't really know the permission of the app. 

This was a first try to be able to run this app the same way on a pod securityenable policy enable cluster or not. But if you prefere we can narrow it down right now. 

Basically I just picked on [other Charts](https://github.com/helm/charts/blob/master/stable/cluster-autoscaler/templates/podsecuritypolicy.yaml)

I agree it is not ideal, it is more for compatibility issue.

Reply:
Yes please narrow it down by disabling: privileged, hostPID, hostIPC, hostNetwork. Thanks

Reply:
@stefanprodan should be good now